.github/workflows/test.yml

```diff
@@ -48,15 +48,21 @@ jobs:
 
           # Check cross-compiling macOS binaries.
           env GOOS=darwin GOARCH=amd64 go build -v ./...
-          env GOOS=darwin GOARCH=arm64 go build -v ./...
 
       - name: go build (Windows, ARM64)
         if: ${{ !startsWith(matrix.go, '1.15.') && !startsWith(matrix.go, '1.16.') }}
         run: |
           # Check cross-compiling Windows binaries.
           env GOOS=windows GOARCH=arm64 go build -v ./...
 
+      - name: go build (macOS, ARM64)
+        if: ${{ !startsWith(matrix.go, '1.15.') }}
```

Comment:
Should we add a comment describing why we don't build for 1.15?

Reply:
Done